com.unity.shadergraph/CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Added
 - Added padding to the shader graph blackboard window to prevent scrollbars from being overlapped by the resize region at the boundaries of the window
 - Support for the XboxSeries platform has been added.
+- Added subshadergraphs for SpeedTree8 shadergraph support: SpeedTree8Wind, SpeedTree8ColorAlpha, SpeedTree8Billboard. Includes a LODCrossfade hlsl that can be used by anything with a LODGroup.
```

Comment:
"Includes a LODCrossfade hlsl that can be used"

A "a LODCrossfade hlsl" what--a script?

Reply:
It's an HLSL shader snippet. It's intended to be included in either handwritten shaders or via a custom function node in Shadergraph.

Reply:
Ok, that's helpful. How is this rewrite?

Includes a HLSL snippet which enables you to use LODCrossfade functionality with any GameObject in an LODGroup.

Reply:
A LODGroup is a component on a prefab or prefab instance. Each LOD contains its own mesh and materials. I guess a prefab instance is technically a type of game object? But it has a LODGroup, not the other way around. 

How about: "Includes an hlsl file that can be included to crossfade between the LODs of any prefab with a LODGroup component."

Reply:
On second thought, I'd like to break the LODCrossfade info into its own line as follows:
"Adds an HLSL file that implements a ShaderGraph-compatible version of the LODDitheringTransition function defined in core."

The functionality in LODCrossfade exists in Common.hlsl in core, however that function uses an HLSL intrinsic called clip(), and doesn't output anything that can be hooked into a shadergraph as a dependency. Because of this, that code gets optimized away and never runs. So this version of LODCrossfade actually is shadergraph specific. Maybe it should be renamed to reflect all this though. Maybe to LODDitheringTransitionSG?

Reply:
Renaming the function sounds like a very good idea. If you do not rename it, this would be my propose rerwite. It is a bit lengthy but precise.
"Adds an HLSL file implementing a version of the Unity core LODDitheringTransition function which is compatible with the Shader Graph package."

If you do rename it, here's my proposed rewrite:
"Adds an HLSL file implementing a version of the Unity core LODDitheringTransition function which is compatible with the Shader Graph package."

How does this sound?

Reply:
Those look the same, but I have renamed it so that's fine! Thank you :)

Reply:
What about "Adds an HLSL file implementing a version of the Unity core LODDitheringTransition function which can be used in a Shader Graph"? It's not the package itself that's the problem, but how shader graphs only take into account things that are connected into the graph.

Reply:
Perfect!